columnphysics/icepack_parameters.F90

```diff
@@ -1103,6 +1119,60 @@ subroutine icepack_init_parameters(   &
          endif
       endif
 
+      !-------------------
+      ! SNICAR SSP table
+      !-------------------
+      if (present(snw_ssp_table_in)     )  snw_ssp_table    = snw_ssp_table_in
```

Comment:
should all of this table allocation code have an 'if (dEdd_snicar)' wrap around it?

Reply:
It probably doesn't matter, but I would say no.  I think anytime a user passes something into icepack_parameters, it should be set that way inside icepack, even if it's not needed.  Adding "if" will just increase complexity and if something is setup improperly, will be harder to debug.